src/pages/[id].page.jsx

```diff
@@ -32,17 +32,29 @@ export default function Details({ details }) {
 
   // dynamic imports need to be asynchronous
   useEffect(() => {
-    const updateCodeBlocks = async () => {
-      const html = await import(`../code-examples/${slug}/html.js`);
-      setHtmlCode(html.default);
+    const updateSandpackExamples = async () => {
+      try {
+        const html = await import(`../code-examples/${slug}/html.js`);
+        setHtmlCode(html.default);
 
-      const css = await import(`../code-examples/${slug}/css.js`);
-      setCssCode(css.default);
+        const css = await import(`../code-examples/${slug}/css.js`);
+        setCssCode(css.default);
 
-      const javascript = await import(`../code-examples/${slug}/javascript.js`);
-      setJavascriptCode(javascript.default);
+        const javascript = await import(
+          `../code-examples/${slug}/javascript.js`
+        );
+        setJavascriptCode(javascript.default);
+
+        // console.log('all code is here');
```

Comment:
Lets remove this comment 🙂

Reply:
gah- I meant to do that. Thanks for catching it! https://github.com/sparkbox/accessible-components/pull/132/commits/b1b9fd41724daf65059e5a6d752d47ded9380ac6